content/en/workflows/service_accounts.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Service Accounts for Workflows
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "/account_management/org_settings/service_accounts/"
+  tag: "Documentation"
+  text: "Learn more about Datadog service accounts"
+---
+
+To add an automated or scheduled trigger to a workflow, the workflow must have an associated service account. The service account is needed to:
+- resolve the connections defined in the workflow actions at runtime
+- provide an identity for workflow executions
+- provide an identity for workflow audit trails
+
+To use automated or scheduled triggers, a workflow must be associated with a unique service account.
+
+## Requirements
+
+To create a service account for a workflow:
+- You must have either the Datadog admin role, or a custom role with the `SERVICE_ACCOUNT_WRITE` permission. For more information on service accounts and permissions, see [Service accounts][1].
+- Before you can add a service account, you need a workflow with at least one action.
+
+## Associate a service account with a workflow
+
+You can dynamically create a service account for your workflow when you [add a trigger][5]. Alternatively, click on the workflow name and add a service account next to **@worklow-** to dynamically create a service account and associate it with the workflow.
```

Comment:
I'm not sure I follow the `**@worklow-**` part. I don't see anything like that in the UI when I click the workflow name: https://drive.google.com/file/d/1rP_BN_N9lqK7mMyaxAdnGUvddweYOc_L/view?usp=sharing